src/main/scala/com/gu/octopusthrift/aws/CloudWatch.scala

```diff
@@ -0,0 +1,49 @@
+package com.gu.octopusthrift.aws
+
+import com.amazonaws.services.cloudwatch.AmazonCloudWatch
+import com.amazonaws.services.cloudwatch.AmazonCloudWatchClientBuilder
+import com.amazonaws.services.cloudwatch.model.Dimension
+import com.amazonaws.services.cloudwatch.model.MetricDatum
+import com.amazonaws.services.cloudwatch.model.PutMetricDataRequest
+import com.amazonaws.services.cloudwatch.model.PutMetricDataResult
+import com.amazonaws.services.cloudwatch.model.StandardUnit
+import com.gu.octopusthrift.Config
+import com.gu.octopusthrift.services.Logging;
+
+object Metrics {
+  val MissingComposerIdOrBodyText = "MissingComposerIdOrBodyText"
+  val FailedThriftConversion = "FailedThriftConversion"
+  val MissingData = "MissingData"
```

Comment:
Maybe worth adding some reference to the payload in the name here, with it representing circumstances when what we're receiving doesn't match the expected payload schema.

Reply:
Great point!